src/pages/AboutMe/AboutMe.tsx

```diff
@@ -39,16 +39,16 @@ const AboutMe: NextPage = () => {
 					/>
 					<AreaTextAboutMe>
 						<H2Head>Sobre Mim </H2Head>
-						<Body1 className="mb-24 mini:mb-16 mt-5">
+						<Body1>
 							Me chamo Kallif Abdon Abrahão, tenho 32 anos,
-							<br /> sou pai de uma menina linda e atuo profissionalmente como
-							desenvolvedor front-end. <br /> Aqui nesse portfólio você pode
-							conhecer mais sobre mim,
+							<br /> sou pai de uma menina linda e atuo profissionalmente <br />
```

Comment:
I recommend that you drop these BR tags altogether. Use <p> wrapping what is meant to be paragraphs and let the text break automatically based on the space available, then you can control the overall width of the text block by adjusting its wrapper's width (body1 or areaTextAboutMe)